CMakeFilters.cmake

```diff
@@ -131,7 +119,7 @@ endif ()
 #-----------------------------------------------------------------------------
 # Option for SzLib support
 #-----------------------------------------------------------------------------
-option (HDF5_ENABLE_SZIP_SUPPORT "Use SZip Filter" OFF)
+option (HDF5_ENABLE_SZIP_SUPPORT "Use SZip Filter" ON)
```

Comment:
This should probably remain as previously

Reply:
Was this change part of other changes or by itself? I can revert the commit or make the individual change; just concerned about getting confused between the state of this between develop and 1.14

Reply:
There were other changes